index.js

```diff
@@ -3,15 +3,18 @@ module.exports = leftpad;
 function leftpad (str, len, ch) {
   str = String(str);
 
-  var i = -1;
-
   if (!ch && ch !== 0) ch = ' ';
 
   len = len - str.length;
+  if (len <= 0) return str;
 
-  while (++i < len) {
-    str = ch + str;
+  ch = ch + '';
+  pad = '';
```

Comment:
`var` must be added, or pad will be a global variable. Don't pollute global variables.